.github/workflows/ci_checks.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup Yarn
         uses: actions/setup-node@v2
         with:
-          node-version: '14'
+          node-version: '16'
```

Comment:
one test failed because `environment.at(-1)` was not recognized by node 14. This is a feature of node 16. But because node 16 is LTS now we can change this.